src/app/zap-templates/zcl/data-model/chip/semantic-tag-namespace-enums.xml

```diff
@@ -44,6 +44,7 @@ TODO: Make these namespace enums global rather than defining them for each clust
 
   <enum name="AreaTypeTag" type="enum8">
     <cluster code="0x0150"/> <!-- Service Area Cluster -->
+    <cluster code="0x0750"/> <!-- Ecosystem Information Cluster -->
```

Comment:
Again, we need to fix this to actually work right...

Reply:
I don't think I will be able to take this on before the July 29th deadline

Reply:
We don't need to, if this is provisional.